inc/commonglpi.class.php

```diff
@@ -984,13 +984,22 @@ class='navicon left'>
          }
          echo "</span>";
 
-         $ma = new MassiveAction([
-               'item' => [
-                  $this->getType() => [
-                     $this->fields['id'] => 1
-                  ]
+         $ma_post = [
+            'item' => [
+               $this->getType() => [
+                  $this->fields['id'] => 1
                ]
-            ],
+            ]
+         ];
+         if (is_subclass_of($this, 'CommonDBChild')) {
```

Comment:
Not sure of this.
`CommonDBChild` already includes a logic which is responsible of checking rights, and it can depends on `$checkParentRights` property.

Reply:
I agree this can cause issue, looking at the code. I saw in the ticket that the issue was related to netrworkports, that is a complex part... Waht was the inital error you talk about?